docs/pages/installation/eks.md

```diff
@@ -6,7 +6,18 @@ permalink: installation/eks
 
 # Quick Start with Amazon Elastic Kubernetes Service (EKS)
 
-## Managed Kubernetes
+Make your way to Meshery and download the token by clicking the "Get Token" option in the dropdown menu. Utilize the token to run the following command:
```

Comment:
I wonder if we can use a moniker for describing where to download the token, like "and download the token by clicking the 'Get Token' option under your User Account avatar."